content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md

```diff
@@ -23,7 +23,7 @@ A clear picture of your state-issued ID is required to complete the identity ver
 ## Tips for uploading or scanning your ID:
 
 * Follow the same tips for taking photo\s of your ID with a camera
-* Save each file as a JPG or PNG. You cannot use a PDF file. Look at the name of your file and it should have an abbreviation after the title (ex: JohnDoeID_Front.jpg) 
```

Comment:
My editor highlights trailing spaces, so I removed a bunch (and some `0x0a` nbsp) which I was here